.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  global:
+    - PORTS="8069:8069"
+    - LINKS="db:db"
+    - DAEMONIZE="1"
+    - ODOO_VERSION="10.0"
+
+  matrix:
+    - TESTS="1" ODOO_VERSION="10.0" DOCKER_CONTEXT="./tests/odoo10"
+    - TESTS="1" ODOO_VERSION="9.0" DOCKER_CONTEXT="./tests/odoo9"
```

Comment:
Activates the tests, which just runs through the binaries in the `tests` folder. There is also `LINT_CHECK` which does the obvious, and `HUB` which pushes to DockerHub